tools/autograd/templates/variable_factories.h

```diff
@@ -16,7 +16,7 @@ namespace torch {
 #define TENSOR(T, S, _1)                                                    \
   inline at::Tensor tensor(                                                 \
       at::ArrayRef<T> values, const at::TensorOptions& options) {           \
-    at::Tensor result = at::tensor(values, options.discard_runtime_type()); \
+    at::Tensor result = at::tensor(values);                                 \
```

Comment:
You'll have to de-variable it here, otherwise we run into the problem of setting a variable's `.data()` to be a variable which breaks. That's why there was `discard_runtime_type()` before. Unless you found it's not necessary?

Reply:
Here we also lost the options?